js/babel-plugin-sprockets-commoner-internal/index.js

```diff
@@ -198,6 +251,25 @@ module.exports = function (context) {
         }
         break;
       }
+    },
+    Identifier: function Identifier(path, state) {
```

Comment:
Just so I make sure I understand, `process` and `Buffer` are being special cased because they are globals in node, correct?